test/unittests/stack_test.cpp

```diff
@@ -80,94 +80,6 @@ TEST(stack, drop_and_peek)
     EXPECT_TRUE(stack.empty());
 }
 
-TEST(stack, clear)
-{
-    Stack<char> stack;
-    stack.push('a');
-    stack.push('b');
-
-    EXPECT_FALSE(stack.empty());
-    EXPECT_EQ(stack.size(), 2);
-
-    stack.drop(0);
-    EXPECT_FALSE(stack.empty());
-    EXPECT_EQ(stack.size(), 2);
-
-    stack.clear();
-    EXPECT_TRUE(stack.empty());
-    EXPECT_EQ(stack.size(), 0);
-}
-
-TEST(stack, resize)
```

Comment:
There's no test for `shrink` besides this.

Reply:
https://github.com/wasmx/fizzy/pull/495/files#diff-931a4a112d70874574895c172f9bfeefL245